vendor/milesight-iot/am103&am103l.js

```diff
@@ -0,0 +1,92 @@
+
+function decodeUplink(input) {
```

Comment:
Please don't use `&` in filenames

Reply:
OK, noted. May I know what's the suggested filename to use if we provide AM103 and AM103L and they share the same codec and decoder?

Reply:
You can do dashes or underscores